package.json

```diff
@@ -46,15 +46,15 @@
     "not op_mini all"
   ],
   "peerDependencies": {
-    "@apmg/mimas": "^0.3.0",
+    "@apmg/mimas": "^1.18",
     "classnames": "^2.2.6",
     "next": "^9.3.0",
     "react": "^16.8.6",
     "react-dom": "^16.8.6",
     "prop-types": "^15.7.2"
   },
   "devDependencies": {
-    "@apmg/mimas": "^0.3.0",
+    "@apmg/mimas": "^1.1.8",
```

Comment:
The versions for mimas should probably match

Reply:
fair point